source/common/json/config_schemas.cc

```diff
@@ -1321,6 +1341,7 @@ const std::string Json::Schema::TOP_LEVEL_CONFIG_SCHEMA(R"EOF(
               "driver" : {
                 "type" : "object",
                 "oneOf" : [
+                  {"$ref" : "#/definitions/xray_driver"},
                   {"$ref" : "#/definitions/lightstep_driver"},
                   {"$ref" : "#/definitions/zipkin_driver"}
                 ]
```

Comment:
Changes to this file are no longer required as this is for v1 only.

Reply:
I thought the next release will be v1.10, is that correct? If it is not required, I will delete this line. Thanks.

Reply:
JSON schemas were used previously. We now rely on Protobufs.